hw2/index-hw2.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Document</title>
+	<link rel="stylesheet" type="text/css" href="style-hw2.css">
+</head>
+<body>
+	<div class="container">
+		<div class="logo">
+			<img src="logo.png" alt="Fitness">
+		</div>
+		<header class="main-header">
+			<nav class="main-menu">
+				<ul class="main-menu__list">
+					<li class="main-menu__item"><a class="main-menu__link" href="#">Главная</a></li>
+					<li class="main-menu__item"><a class="main-menu__link" href="#">Товары</a></li>
+					<li class="main-menu__item"><a class="main-menu__link" href="#">О_нас</a></li>
+					<li class="main-menu__item"><a class="main-menu__link" href="#">Контакты</a></li>
+				</ul>
+			</nav>
+		</header>
+		<aside class="products">
+			<ul class="products__list">
+				<li class="products__item"><a class="products__link" href="#">Ноутбуки</a></li>
+				<li class="products__item"><a class="products__link" href="#">Планшеты</a></li>
+				<li class="products__item"><a class="products__link" href="#">Телефоны</a></li>
+				<li class="products__item"><a class="products__link" href="#">Телевизоры</a></li>
+				<li class="products__item"><a class="products__link" href="#">Бытовая техника</a></li>
+				<li class="products__item"><a class="products__link" href="#">Автотовары</a></li>
+			</ul>
+		</aside>
+		<main class="main-info">
+			<div class="main-info__item">
+				<img class="main-info__image" src="main-image.png" alt="bicycle">
+			</div>
+			<div class="main-info__text">
+				Lorem Ipsum - це текст-"риба", що використовується в друкарстві та дизайні. Lorem Ipsum є, фактично, стандартною "рибою" аж з XVI сторіччя, коли невідомий друкар взяв шрифтову гранку та склав на ній підбірку зразків шрифтів. "Риба" не тільки успішно пережила п'ять століть, але й прижилася в електронному верстуванні, залишаючись по суті незмінною. Вона популяризувалась в 60-их роках минулого сторіччя завдяки виданню зразків шрифтів Letraset, які містили уривки з Lorem Ipsum, і вдруге - нещодавно завдяки програмам комп'ютерного верстування на кшталт Aldus Pagemaker, які використовували різні версії Lorem Ipsum.
+			</div>
+		</main>
+		<div class="clear"></div>
+		<footer class="main-footer">
+			<div class="footer-menu">
```

Comment:
Do you think you need this wrapping div all around each UL? can we eliminate it?